### ISSUE TITLE
Bump to v25211

### ISSUE THREAD

**Issue by YT-Advanced** [HuskyDG/magisk-files#104]
In the title

**Comment by YT-Advanced** [comment]
> I will bump to v26 soon after major exams completed

ơ bác cũng đang đi học hả :))) Em cũng đang ôn thi chuẩn bị vào lớp 10 :(

**Comment by jont4** [comment]
> > I will bump to v26 soon after major exams completed
> 
> Ok, how long does it take?

as long as it takes

**Comment by JeelsBoobz** [comment]
> I will bump to v26 soon after major exams completed

Need update quickly, Shamiko v0.6 not running perfectly if update available
I don't know if Shamiko update, old module wil be crash 😔

**Comment by Nitroandaccounts** [comment]
Yay! Thanks!

**Comment by Jordan231111** [comment]
> The next release will come on May. Please wait!

Can you check nox android 12 and try to support that, I think there just some compatibility issue based on the logs and not disk permissions error

**Comment by Jordan231111** [comment]
[magisk_log_2023-04-30T13.15.06.log](https://github.com/HuskyDG/magisk-files/files/11361882/magisk_log_2023-04-30T13.15.06.log)

**Comment by skelletR** [comment]
> > The next release will come on May. Please wait!
> 
> Can you check nox android 12 and try to support that, I think there just some compatibility issue based on the logs and not disk permissions error

I have the same problem, try it again with the root enabled permanent, on nox settings not uncheck after install magisk Delta

**Comment by Jordan231111** [comment]
> > > The next release will come on May. Please wait!
> > 
> > 
> > Can you check nox android 12 and try to support that, I think there just some compatibility issue based on the logs and not disk permissions error
> 
> I have the same problem, try it again with the root enabled permanent, on nox settings not uncheck after install magisk Delta, but still can't work the lsposed riru with riru 25

The main issue is magisk root in general, not just riru. Also for nox I already did all the tests that you mentioned one of them. All we can do is wait for him to finish his exams and for him to confirm that the issue is with magisk or if it is nox..

**Comment by skelletR** [comment]
> > > > The next release will come on May. Please wait!
> > > 
> > > 
> > > Can you check nox android 12 and try to support that, I think there just some compatibility issue based on the logs and not disk permissions error
> > 
> > 
> > I have the same problem, try it again with the root enabled permanent, on nox settings not uncheck after install magisk Delta, but still can't work the lsposed riru with riru 25
> 
> The main issue is magisk root in general, not just riru. Also for nox I already did all the tests that you mentioned one of them. All we can do is wait for him to finish his exams and for him to confirm that the issue is with magisk or if it is nox..

Understand, just say because i have root access on nox 12 with this option

**Comment by Fabali** [comment]
> 大佬，都5月了，还更新吗？₍˄·͈༝·͈˄*₎◞ ̑̑

**Comment by magojohnji** [comment]
It is May.👶

**Comment by KodeXc** [comment]
May has 31 days. Let's wait patiently.

**Comment by xiaofengv233** [comment]
抱歉，出了点问题。

**Comment by XY50** [comment]
还更新吗？5月都快完了

**Comment by Fabali** [comment]
> 

大概率摆烂了

**Comment by magojohnji** [comment]
> 抱歉，出了点问题。

What's wrong?

**Comment by KodeXc** [comment]
Still within May.. as excited as we are, if the current version works, no reason to chase for something that you're getting for free. Also, ICYMI:

"Hello! It has been more than month that Magisk Delta is inactive. Because I am the only one maintain this fork. Also school is my priority and I have fo focus in studying. During that time, the study takes my time with many imprortant subprojects so... 😅
Final words, Delta has merged v26.1 changes, but need some adjustments before being released. So wait for it!"

**Comment by foxhacker0000** [comment]
26 not released yet

you told in May month

**Comment by MustardChef** [comment]
> 26 not released yet
> 
> you told in May month

Sure, but you still have to be patient.

**Comment by foxhacker0000** [comment]
thank you sir but how much time?
also i am your fan and supporting you in my telegram channel :)

**Comment by MastaG** [comment]
While I don't need to update as v25 provides everything I need. I wonder if husky is alright?
We didnt hear from you for a while.

**Comment by MastaG** [comment]
I don't think so, I also rely on Delta's MagiskHide + Displax's SafetyNet fix MOD (https://github.com/Displax/safetynet-fix) + Shamiko v0.6 (https://github.com/LSPosed/LSPosed.github.io/releases) newer releases require magisk v26.x...

To keep Microsoft apps from flagging my device.
Vanilla Magisk just doesn't cut it for some reason.

I think it's because when using Magisk Delta, the "hide" + "enforce sulist" options allows you to only have "su" visible for the apps -you- choose.

With vanilla magisk, an app can probably spawn some sneaky process which detects "su" in some way...

**Comment by yaomoshen5200** [comment]
After using the magic delta root, I started Frida Hook on my phone. How to solve the problem? Using magic is possible, but using magic delta fails. I want to know how to set it, but even if I can't catch packets, how to set it up

**Comment by yaomoshen5200** [comment]
@HarriBuh 

What does it mean to explain the white dot? How to solve it

**Comment by magojohnji** [comment]
> Since I've run out off patience with this one: Does anybody here know if there was an actively maintained alternative app to hide Magisk? I've done some searching, but couldn't find anything.
Since Magisk Delta itself met some problems, I think if we need newer Magisk urgently, we can use something to temporarily replace till Magisk Delta get well.
Here's a module : https://github.com/HuskyDG/MagiskHide
It can allow you run MagiskHide with official Magisk.
But hope Magisk Delta can get well ASAP.

**Comment by magojohnji** [comment]
> Since I've run out off patience with this one: Does anybody here know if there was an actively maintained alternative app to hide Magisk? I've done some searching, but couldn't find anything.

Since Magisk Delta itself met some problems, I think if we need newer Magisk urgently, we can use something to temporarily replace till Magisk Delta get well.
Here's a module : https://github.com/HuskyDG/MagiskHide
It can allow you run MagiskHide with official Magisk.
But hope Magisk Delta can get well ASAP.

**Comment by magojohnji** [comment]
> Since I've run out off patience with this one: Does anybody here know if there was an actively maintained alternative app to hide Magisk? I've done some searching, but couldn't find anything.

Since Magisk Delta itself met some problems, I think if we need newer Magisk urgently, we can use something to temporarily replace till Magisk Delta gets well.
Here's a module : https://github.com/HuskyDG/MagiskHide
It can allow you run MagiskHide with official Magisk.
But hope Magisk Delta can get well ASAP.

**Comment by TheElectrinix** [comment]
> > Since I've run out off patience with this one: Does anybody here know if there was an actively maintained alternative app to hide Magisk? I've done some searching, but couldn't find anything.
> 
> Since Magisk Delta itself met some problems, I think if we need newer Magisk urgently, we can use something to temporarily replace till Magisk Delta gets well. Here's a module : https://github.com/HuskyDG/MagiskHide It can allow you run MagiskHide with official Magisk. But hope Magisk Delta can get well ASAP.

I also wish the same. I'm waiting for the magisk delta 26 for updating my phone because newer versions of the os makes someting that the selinux modules do not charge, and it's patched in the magisk 26 version. Also, thanks HuskyDG for making the best magisk version ever.

**Comment by MastaG** [comment]
Yeah, a sign of life would be nice

**Comment by TheElectrinix** [comment]
I would like even to participate in a alpha test of the v26, now that I founded a way of how to unbrick my phone, I would carry with the responsability of trying unfinished software.

**Comment by XY50** [comment]
啥时候更新啊？再不更新的话，我就换回官方版的了

**Comment by Fabali** [comment]
> 啥时候更新啊？再不更新的话，我就换回官方版的了
好像在电报里面说不更新了，作者转去ksu了，我记得

**Comment by jont4** [comment]
> > 啥时候更新啊？再不更新的话，我就换回官方版的了
> > 好像在电报里面说不更新了，作者转去ksu了，我记得

English please

**Comment by Fabali** [comment]
> 

The author seems to have stopped updating and started using KSU. This message was mentioned on Telegram.

**Comment by TheElectrinix** [comment]
> > > 啥时候更新啊？再不更新的话，我就换回官方版的了
> > > 好像在电报里面说不更新了，作者转去ksu了，我记得
> 
> English please
Translation: When will it be updated? If It don't get updated, I'll switch back to the official version

**Comment by TheElectrinix** [comment]
> > 
> 
> The author seems to have stopped updating and started using KSU. This message was mentioned on Telegram.

share the telegram link please

**Comment by Fabali** [comment]
> > > 
> > 
> > 
> > The author seems to have stopped updating and started using KSU. This message was mentioned on Telegram.
> 
> share the telegram link please

https://t.me/+2l5jXxdvU5YwNjZl

**Comment by TheElectrinix** [comment]
> > > > 
> > > 
> > > 
> > > The author seems to have stopped updating and started using KSU. This message was mentioned on Telegram.
> > 
> > 
> > share the telegram link please
> 
> https://t.me/+2l5jXxdvU5YwNjZl

thx

**Comment by SkipM4** [comment]
So just forget it
![Screenshot.png](https://github.com/HuskyDG/magisk-files/assets/40311581/e79f7baf-b4c1-433e-ba75-058e0d81f0ce)

**Comment by MastaG** [comment]
Alright

**Comment by jont4** [comment]
He should at least warn here, make an official announcement. Not leaving us waiting for an upgrade without any warning that had abandoned the Delta.

**Comment by KodeXc** [comment]
lol. all these entitled people.

**Comment by YT-Advanced** [comment]
Finnally .-.

**Comment by magojohnji** [comment]
Anyway, it's fine in the end. 🎉🎉🎉
